### PR TITLE
FA-125 Fragment App Navigation

### PR DESCRIPTION
**Pull request by brosscode**
Adds Navigation to the fragments.

### CODE REVIEW

File: appFragments/src/main/java/com/brandon/appfragments/di/NavigationModule.kt [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+import com.brandon.appfragments.navigation.FragmentNavImpl
+import com.brandon.navigation.FragmentNav
+import dagger.Module
+import dagger.Provides
+import dagger.hilt.InstallIn
+import dagger.hilt.android.components.ViewModelComponent
+
+@InstallIn(ViewModelComponent::class)
+@Module
+class NavigationModule {
```
Comment (ForzenAcademy) [inline appFragments/src/main/java/com/brandon/appfragments/di/NavigationModule.kt]: object

File: appFragments/src/main/java/com/brandon/appfragments/view/LandingPageFragment.kt [hunk 2]
```diff
@@ -0,0 +1,6 @@
+package com.brandon.appfragments.view
+
+import androidx.fragment.app.Fragment
+
+class LandingPageFragment: Fragment() {
```
Comment (ForzenAcademy) [inline appFragments/src/main/java/com/brandon/appfragments/view/LandingPageFragment.kt:5]: Put a todo here

File: appFragments/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -0,0 +1 @@
+<resources></resources>
```
Comment (ForzenAcademy) [inline appFragments/src/main/res/values/strings.xml]: Remove this file

File: loginFragments/src/main/java/com/brandon/loginfragments/viewmodel/FragmentLoginViewmodel.kt [hunk 4]
```diff
@@ -13,10 +15,9 @@ import javax.inject.Inject
 class FragmentLoginViewmodel @Inject constructor(
     override val loginUseCase: LoginUseCase,
     override val connectivityManager: ConnectivityManager,
+    private val navigation: FragmentNav
```
Comment (ForzenAcademy) [inline loginFragments/src/main/java/com/brandon/loginfragments/viewmodel/FragmentLoginViewmodel.kt]: Always use trailing commas

**Review by ForzenAcademy — APPROVED** [review]

File: appFragments/src/main/java/com/brandon/appfragments/di/NavigationModule.kt (hunk 1 above)
Comment (brosscode) [reply appFragments/src/main/java/com/brandon/appfragments/di/NavigationModule.kt]: objected it

File: appFragments/src/main/java/com/brandon/appfragments/view/LandingPageFragment.kt (hunk 2 above)
Comment (brosscode) [reply appFragments/src/main/java/com/brandon/appfragments/view/LandingPageFragment.kt:5]: Added a TODO

File: appFragments/src/main/res/values/strings.xml (hunk 3 above)
Comment (brosscode) [reply appFragments/src/main/res/values/strings.xml]: Removed

File: loginFragments/src/main/java/com/brandon/loginfragments/viewmodel/FragmentLoginViewmodel.kt (hunk 4 above)
Comment (brosscode) [reply loginFragments/src/main/java/com/brandon/loginfragments/viewmodel/FragmentLoginViewmodel.kt]: trailing comma

**Pull request merged**